ngraph/core/include/ngraph/op/prior_box.hpp

```diff
@@ -9,10 +9,11 @@
 
 namespace ngraph {
 namespace op {
-using PriorBoxAttrs = ov::op::v0::PriorBox::Attributes;
```

Comment:
Please revert this change. IIt should allow to fix opencv

Reply:
updated in commit 7b99a21